src/wasm.rs

```diff
@@ -423,32 +444,101 @@ pub enum Instruction {
     F64Ceil,
     F64Nearest,
 
-    I32Load { flags: u32, offset: u32 },
-    I32Store { flags: u32, offset: u32 },
-    I32Load8S { flags: u32, offset: u32 },
-    I32Load8U { flags: u32, offset: u32 },
-    I32Store8 { flags: u32, offset: u32 },
-    I32Load16S { flags: u32, offset: u32 },
-    I32Load16U { flags: u32, offset: u32 },
-    I32Store16 { flags: u32, offset: u32 },
-
-    I64Load { flags: u32, offset: u32 },
-    I64Store { flags: u32, offset: u32 },
-    I64Load8S { flags: u32, offset: u32 },
-    I64Load8U { flags: u32, offset: u32 },
-    I64Store8 { flags: u32, offset: u32 },
-    I64Load16S { flags: u32, offset: u32 },
-    I64Load16U { flags: u32, offset: u32 },
-    I64Store16 { flags: u32, offset: u32 },
-    I64Load32S { flags: u32, offset: u32 },
-    I64Load32U { flags: u32, offset: u32 },
-    I64Store32 { flags: u32, offset: u32 },
-
-    F32Load { flags: u32, offset: u32 },
-    F32Store { flags: u32, offset: u32 },
-
-    F64Load { flags: u32, offset: u32 },
-    F64Store { flags: u32, offset: u32 },
+    I32Load {
```

Comment:
I find the new formatting here a bit worse. Any way to exclude this enum?

Reply:
Should be fixed!